packages/account-v2/package.json

```diff
@@ -15,7 +15,7 @@
         "@deriv/api": "^1.0.0",
         "@deriv/library": "^1.0.0",
         "@deriv/quill-design": "^1.3.2",
-        "@deriv/quill-icons": "^1.17.4",
+        "@deriv/quill-icons": "^1.17.25",
```

Comment:
why are we updating the version here?

Reply:
the PR was failing because of it :( so i had to update it here